cli/command/plugin/remove.go

```diff
@@ -36,17 +37,13 @@ func newRemoveCommand(dockerCli command.Cli) *cobra.Command {
 }
 
 func runRemove(ctx context.Context, dockerCli command.Cli, opts *rmOptions) error {
-	var errs cli.Errors
+	var errs error
 	for _, name := range opts.plugins {
 		if err := dockerCli.Client().PluginRemove(ctx, name, types.PluginRemoveOptions{Force: opts.force}); err != nil {
-			errs = append(errs, err)
+			errs = errors.Join(errs, err)
```

Comment:
This is not the same as the previous one, as it always produce `*joinError` with `[]error` of length 2. The first error is of type `*errors.joinError` which combines all of the previous errors.

Reply:
The correct way to use `errors.Join` in this case is to first collect errors into `errs []error` and then `return errors.Join(errs...)`